_ont/ont-zte-f6005v3.md

```diff
@@ -0,0 +1,341 @@
+---
+title: ZTE F6005v3 
+has_children: false
+layout: default
+parent: ZTE
+---
+
+# Hardware Specifications
+
+|              |                                                                   |
+| ------------ | ----------------------------------------------------------------- |
+| Vendor/Brand | ZTE                                                               |
+| Model        | F6005v3                                                           |
+| ODM          | ✅                                                                |
+| CPU          | ZTE ZX279133@A53                                                  |
+| CPU Clock    | 2x1200MHz                                                         |
+| Chipset      | ZTE ZX279133                                                      |
+| Flash        | 128 MB (SPI NAND FM25S01A)                                        |
+| RAM          | 128 MB                                                            |
+| System       | Customized Linux by ZTE                                           |
+| 2.5GBaseT    | No                                                                |
```

Comment:
```suggestion
| 2.5GBaseT    | Yes                                                               |
```

Reply:
My fault, I've copied F601 template and missed this one 😅

I've some other fresh news, so I'll fix on the next pull. Thanks 😊

Reply:
Thank you for your research!